runner/src/runner/pipeline.clj

```diff
@@ -32,9 +33,14 @@
                    :run-id run-id
                    :line line}]]))
 
+(defn log->queue
+  [producer content]
+  (ev/publish producer content))
+
 (defn log-event
-  [db-client run-id line]
-  (log->db db-client run-id (str "[bob] " line)))
+  [db-client queue-chan {:keys [message run-id] :as content}]
+  (log->db db-client run-id (str "[bob] " message))
+  (log->queue queue-chan content))
```

Comment:
can call publish here directly?

Reply:
Also we don't have to store the events in the db i think.

Reply:
Yeah keep it the way it was. Do the logging to db and events separately.

Reply:
You think I should make them two different calls? I thought it should be more or less the same, I don't get the difference so much.

Reply:
done